--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -243,9 +243,9 @@
 
 option(LOADER_CODEGEN "Enable vulkan loader code generation")
 if(LOADER_CODEGEN)
-    find_package(Python3 REQUIRED)
+    find_package(Python REQUIRED)
     add_custom_target(loader_codegen
-        COMMAND Python3::Interpreter ${PROJECT_SOURCE_DIR}/scripts/generate_source.py
+        COMMAND Python::Interpreter ${PROJECT_SOURCE_DIR}/scripts/generate_source.py
             "${VULKAN_HEADERS_INSTALL_DIR}/${CMAKE_INSTALL_DATADIR}/vulkan/registry"
             --generated-version ${VulkanHeaders_VERSION} --incremental --api ${API_TYPE}
     )
